docs/courses/exercises/multiple-choice-exercises/questions.md

```diff
@@ -0,0 +1,73 @@
+# Writing `MultipleChoiceExercise` and `PureMultipleChoiceExercise` Questions
+
+This page provides guidance on writing good multiple choice questions.
+
+## Examples
+
+From [Understanding your data](https://campus.datacamp.com/courses/deep-learning-in-python/building-deep-learning-models-with-keras?ex=2) in "Deep Learning in Python". This provides context around the workflow (you should explore your dataset before you start modeling it), and requires that the students type some code, and actually look at the dataset in order to answer.
```

Comment:
Good example but the length of the assignment text here violates the content guidelines :(

Reply:
I added a warning about this exceeding guidelines via https://github.com/datacamp/authoring/commit/9dedae687bf6ebdf48db8fc6b70047351af29640

If you have examples of other good MC questions, I'm happy to swap them in.